src/main/resources/db/changelog/changes/06-create-categories-table.yml

```diff
@@ -0,0 +1,30 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-categories-table
+      author: den
+      changes:
+        - createTable:
+            tableName: categories
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: description
+                  type: varchar(255)
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```

Comment:
make sure each file ends with exactly one empty line to avoid this
![image](https://github.com/DenysMaksymKachalup/spring-bookstore/assets/49767049/14dba767-1382-4977-97d5-b11f99102599)